src/main/resources/avro/beacon.avdl

```diff
@@ -137,11 +137,8 @@ record BeaconInformationResource {
 The response to the Beacon query
 */
 record ResponseResource {
-  /** Whether the beacon has observed variants. True if an observation exactly matches request. Overlap if an
-  observation overlaps request, but not exactly, as in the case of indels or if the query used wildcard for
-  allele. False if data are present at the requested position but no observations exactly match or overlap. Null
-  otherwise. */
-  string exists;
+  /** Indicator of whether the beacon has observed the variant. */
+  union{ null, boolean } exists;
```

Comment:
We're taking null as "not specified" here, e.g. when you want to return an error (invalid query etc.). It's not meant to be a "valid" response. Now that you mention this, it probably makes sense to change the way errors are reported to avoid this confusion.